spring-cloud-common-security-config/spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/OAuthSecurityConfiguration.java

```diff
@@ -155,59 +106,13 @@ public class OAuthSecurityConfiguration {
 	@Autowired
 	protected LogoutSuccessHandler logoutSuccessHandler;
 
-	protected OpaqueTokenIntrospector opaqueTokenIntrospector;
-
-	protected ProviderManager providerManager;
-
-	public AuthorizationProperties getAuthorizationProperties() {
-		return authorizationProperties;
-	}
-
-	public void setAuthorizationProperties(AuthorizationProperties authorizationProperties) {
-		this.authorizationProperties = authorizationProperties;
-	}
-
-	public OpaqueTokenIntrospector getOpaqueTokenIntrospector() {
-		return opaqueTokenIntrospector;
-	}
-
 	@Autowired(required = false)
-	public void setOpaqueTokenIntrospector(OpaqueTokenIntrospector opaqueTokenIntrospector) {
-		this.opaqueTokenIntrospector = opaqueTokenIntrospector;
-	}
-
-	public ProviderManager getProviderManager() {
-		return providerManager;
-	}
-
-	@Autowired(required = false)
-	public void setProviderManager(ProviderManager providerManager) {
-		this.providerManager = providerManager;
-	}
-
-	public OAuth2ResourceServerProperties getoAuth2ResourceServerProperties() {
-		return oAuth2ResourceServerProperties;
-	}
-
-	public void setoAuth2ResourceServerProperties(OAuth2ResourceServerProperties oAuth2ResourceServerProperties) {
-		this.oAuth2ResourceServerProperties = oAuth2ResourceServerProperties;
-	}
-
-	public SecurityStateBean getSecurityStateBean() {
-		return securityStateBean;
-	}
-
-	public void setSecurityStateBean(SecurityStateBean securityStateBean) {
-		this.securityStateBean = securityStateBean;
-	}
-
-	protected HttpBasicConfigurer configure(HttpSecurity http) throws Exception {
+	protected ProviderManager providerManager;
 
-		final RequestMatcher textHtmlMatcher = new MediaTypeRequestMatcher(
-				new BrowserDetectingContentNegotiationStrategy(),
-				MediaType.TEXT_HTML);
+	@Bean
+	public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
```

Comment:
Is this bean being used?   I see we have a SecurityFilterChain for both Skipper and SCDF.

Reply:
Mentioned in other comment, dead class.